### PR TITLE
Add img prop for AssetScanner component

### PR DESCRIPTION
**Pull request by mrtraser**
Give user ability to provide a custom image to be recognized

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-326.gearboxjs.preview.cogniteapp.com

File: src/components/AssetScanner/stories/AssetScanner.stories.tsx [hunk 1]
```diff
@@ -11,6 +11,43 @@ import full from './full.md';
 import customNotificationsDoc from './notifications.md';
 import ocrRequestDoc from './ocr-request.md';
 
+const FileInputComponent = (props: any) => {
+  const [image, setImage] = useState('');
+
+  const onChange = async (e: SyntheticEvent) => {
```
Comment (xitman78) [inline src/components/AssetScanner/stories/AssetScanner.stories.tsx]: this should fix TS error 2 lines below
```suggestion
  const onChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment (mrtraser) [reply src/components/AssetScanner/stories/AssetScanner.stories.tsx]: Nope, it doesn't help to avoid TS errors

**Comment by mrtraser** [comment]
Closes #318

**Pull request merged**